processor/spanmetricsprocessor/processor.go

```diff
@@ -577,3 +580,16 @@ func setExemplars(exemplarsData []exemplarData, timestamp pcommon.Timestamp, exe
 
 	es.CopyTo(exemplars)
 }
+
+// buildMetricName builds the metric name from the namespace and metric name.
+// if normalized is true, the metric name will be normalized concatenating the namespace and metric name with a dot.
+// if normalized is false, the metric name will be concatenated with an underscore.
+func buildMetricName(namespace string, metricName string, normalized bool) string {
+	if namespace != "" {
+		if normalized {
+			return namespace + "." + metricName
```

Comment:
Maybe we shall avoid string concatenation if `namespace` is empty.

Reply:
Yes, currently `namesapce` will concatenation only if it is not empty, otherwise it returns directly

Reply:
Oh, indeed, sorry :)